src/models/tag/traits/collection.rs

```diff
@@ -13,6 +13,11 @@ use crate::{
 
 use crate::models::tag::TagDetails;
 
+const CACHE_SORTED_SET_PREFIX: &str = "Cache:Sorted";
+pub const CACHE_SET_PREFIX: &str = "Cache";
+// Minutes
+const CACHE_TTL: i64 = 180;
```

Comment:
Best if seconds.

We are not creating `CACHE_EXPIRATION` constants in the other `CACHE` implementations ( that is, `recommeneded` user streams). Maybe we should unify patterns: either create constants for all of them or hardcode the value within the function. Creating constants is better.